.github/actions/build-push-pr.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+cat <<EOF > local.mk
+PUBLISH_URL=https://api.synocommunity.com
+PUBLISH_API_KEY=
+MAINTAINER?=
+MAINTAINER_URL=
+DISTRIBUTOR=SynoCommunity
+DISTRIBUTOR_URL=https://synocommunity.com/
+REPORT_URL=https://github.com/SynoCommunity/spksrc/issues
+SUPPORT_URL=https://github.com/SynoCommunity/spksrc/issues
+DEFAULT_TC=6.1
+PARALLEL_MAKE=4
+EOF
+
```

Comment:
I suppose it would be better to call `make setup` instead.
With `make setup` only `DEFAULT_TC=6.1` is overwritten when the file `local.mk` already exists. All other settings will not be changed in an existing `local.mk`
`make setup` is based on two other make targets:
- `local.mk` to create a default local.mk when the file does not exist.
- `dsm-%` to set DEFAULT_TC to the given value

Is is definitly bad when this script overwrites manually added settings like `PUBLISH_API_KEY`

Reply:
Thanks good call!

PUBLISH_API_KEY should not be committed to the git repository...🧐 Github secrets should be used instead.

Reply:
To actually publish one needs to uncomment the API_KEY and add publish- to the GH_ARCH environment variable.

https://github.com/SynoCommunity/spksrc/blob/5a6bae603e4dd3bd37c09acee9c4ca09bf0f07d4/.github/workflows/build.yml#L67-L70